yt/utilities/lib/geometry_utils.pxd

```diff
@@ -11,12 +11,21 @@ cimport numpy as np
 from libc.float cimport DBL_MANT_DIG
 from libc.math cimport frexp, ldexp, sqrt
 
-DEF ORDER_MAX=20
-DEF INDEX_MAX_64=2097151
+
```

Comment:
Thanks for fixing this -- but to be honest I don't see this example of removing a DEF as a net improvement for readability.

Reply:
Looks like there may be a better solution in the next pre-release, see https://github.com/cython/cython/pull/5242